mteb/tasks/Audio/Any2AnyRetrieval/JamAltArtist.py

```diff
@@ -0,0 +1,232 @@
+from __future__ import annotations
+
+from collections import defaultdict
+
+import datasets
+
+from mteb.abstasks.Audio.AbsTaskAny2AnyRetrieval import AbsTaskAny2AnyRetrieval
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class JamAltArtist(AbsTaskAny2AnyRetrieval):
+    metadata = TaskMetadata(
+        name="JamAltArtistA2ARetrieval",
+        description="Given audio clip of a song (query), retrieve all songs from the same artist in the Jam-Alt-Lines dataset",
+        reference="https://huggingface.co/datasets/jamendolyrics/jam-alt-lines",
+        dataset={
+            "path": "jamendolyrics/jam-alt-lines",
+            "revision": "main",
```

Comment:
We should also specify the revision (a hash, not main) and the subset, e.g `"pure"`.